doc/Extensions/Serialization/HowTo-Serialization.md

```diff
@@ -0,0 +1,123 @@
+# How-To: Serialize and Deserialize JSON Data
+
+Accessing the serialized and deserialized representation of an object can be important for dynamic, data-rich applications. Uno.Extensions supports the [new](https://devblogs.microsoft.com/dotnet/try-the-new-system-text-json-source-generator) serialization technique powered by code generation, but you can optionally revert to the previous one which uses reflection.
+
+## Step-by-steps
+
+### 1. Opt into Serialization
+
+* Call the `UseSerialization()` method to register a serializer that implements `ISerializer` with the service collection:
+
+```csharp
+    private IHost Host { get; }
```

Comment:
![image](https://user-images.githubusercontent.com/2359550/168687005-695f25cb-f1bb-4677-9093-67fb420950d8.png)

1. for code snippets belonging to a `*` bullet item, indent them 1 tier deeper than the bullet to align them
2. dont start with +1 indent level within the code snippets

\#2 collides with \#1, so you just need to adjust the indent of: \`\`\`